.actor/INPUT_SCHEMA.json

```diff
@@ -0,0 +1,90 @@
+{
+    "title": "Snowflake uploader input",
+    "type": "object",
+    "schemaVersion": 1,
+    "properties": {
+        "synchronizeSchema": {
+            "title": "Database schema columns",
+            "type": "array",
+            "description": "SETTING THIS OPTION WILL DROP YOUR TABLE. In case your destination table has a schema incompatible with the dataset, you can provide a list of column names and their types, then the table will be synchronized according to your definitions by dropping and recreating the table. Key is the column name, value is the column type (as in SQL query, like VARCHAR). This is optional if your table already has the desired schema.",
+            "editor": "keyValue"
+        },
+        "transformJsonKeyFunction": {
+            "title": "Transform JSON key function",
+            "type": "string",
+            "description": "Enter body of a function that will transform JSON keys of dataset objecs. You have access to 'key' argument and must return a string. Example: 'return key.toLowerCase()'",
+            "editor": "textfield"
+        },
```

Comment:
1. I think it is more flexible to just provide the whole dataset item as an argument. Just key adjustment could be useful as well for people who cannot code much but I would skip it for now until we have users that want that.
2. We have a "editor" JavaScript, see here for example https://github.com/metalwarrior665/actor-dedup-datasets/blob/master/INPUT_SCHEMA.json#L66